css/_vertical_filmstrip_overrides.scss

```diff
@@ -69,10 +80,11 @@
             flex: 1;
             flex-direction: column;
             height: auto;
-            overflow-x: hidden !important;
+            justify-content: flex-end;
 
             .remote-videos-container {
-                flex-direction: column;
+                flex-direction: column-reverse;
+                overflow-x: hidden !important;
             }
```

Comment:
What let you have vertical filmstrip scrolling start from the bottom of the filmstrip?

Reply:
I'm noticing an issue right now on firefox with vertical filmstrip where the videos do not scroll and instead keep stacking on each other until they're pushing out of the window. I'm wondering if it's related to the style changes to get the scrolling to start from the bottom of the filmstrip.

Reply:
I'll check that. The thing I did to make the scroll work from bottom is to set column-reverse to the inner container and move the overflow-x from the parent to the inner container as well. The set the flex-end to the outer container.